src/libteam/0005-libteam-Add-warm_reboot-mode.patch

```diff
@@ -0,0 +1,354 @@
+diff --git a/libteam/ifinfo.c b/libteam/ifinfo.c
+index 72155ae..266cc21 100644
+--- a/libteam/ifinfo.c
++++ b/libteam/ifinfo.c
+@@ -58,6 +58,7 @@ struct team_ifinfo {
+ 	char			ifname[IFNAMSIZ];
+ 	uint32_t		master_ifindex;
+ 	bool			admin_state;
++        bool                    orig_addr_updated; /* FIXME: Check this. I think we don't need this flag */
```

Comment:
>         [](start = 1, length = 8)

Better follow existing code's indentation style.

Reply:
this chunk was removed. It was needed only if netdev wasn't removed